File: SCALA/proplems.scala
object Arithmetic {
  def add(a: Int, b: Int): Int = a + b
  def subtract(a: Int, b: Int): Int = a - b
  def multiply(a: Int, b: Int): Int = a * b
  def divide(a: Int, b: Int): Option[Int] = if (b != 0) Some(a / b) else None // Add error handling for division by zero

  def main(args: Array[String]): Unit = {
    val x = 10
    val y = 5
    println(s"Add: ${add(x, y)}")
    println(s"Subtract: ${subtract(x, y)}")
    println(s"Multiply: ${multiply(x, y)}")
    println(s"Divide: ${divide(x, y).getOrElse("Error (division by zero)")}")
  }
}

File: JULIA/proplems.jl
function add(a, b) a + b end
function subtract(a, b) a - b end
function multiply(a, b) a * b end
function divide(a, b) b != 0 ? a / b : "Error (division by zero)" end

x = 10
y = 5

println("Add: ", add(x, y))
println("Subtract: ", subtract(x, y))
println("Multiply: ", multiply(x, y))
println("Divide: ", divide(x, y))
